code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -273,9 +271,10 @@
 /mob/living/simple_animal/gib()
 	if(icon_gib)
 		flick(icon_gib, src)
-	if(meat_amount && meat_type)
-		for(var/i = 0; i < meat_amount; i++)
-			new meat_type(src.loc)
+	if(butcher_results)
+		for(var/path in butcher_results)
+			for(var/i = 1; i <= butcher_results[path];i++)

```

Comment:
Doesn't this mean that this will double the amount of meat produced?

That is to say...you harvest something with a knife---it makes the meat, then it gibs and does this all over again.